CHANGELOG.md

```diff
@@ -49,7 +138,7 @@
  * [[`b9117a001b`](https://github.com/level/levelup/commit/b9117a001b)] - fixes the leveldb link (Manuel Ernst)
  * [[`3d541dbfc9`](https://github.com/level/levelup/commit/3d541dbfc9)] - remove 0.8 from travis (Rod Vagg)
 
-### 0.19.0 @ Aug 26 2014
+## 0.19.0
```

Comment:
Dates are useful, put version numbers into historical context

Reply:
I'll fix. I removed it since the commits themselves give away the timeline, but I can revert this, np.